packages/realm-server/migrations/1712771547705_initial.js

```diff
@@ -1,7 +1,7 @@
 exports.up = (pgm) => {
   pgm.createTable('indexed_cards', {
     card_url: { type: 'varchar', notNull: true },
-    realm_version: { type: 'varchar', notNull: true },
+    realm_version: { type: 'integer', notNull: true },
```

Comment:
I opted to adjust this migration instead of making a new one since we don't actually have any data yet to migrate